src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,58 +4,71 @@
     "origin.toomanyorigins.hare.name": "Hare",
 
     "origin.toomanyorigins.hisskin.name": "Hiss-kin",
-    "origin.toomanyorigins.hisskin.description": "Creeper'ın bu uzak akrabası patlamalarını kaçış taktiği olarak kullanır.",
+    "origin.toomanyorigins.hisskin.description": "Creeperın bu uzak akrabası, patlamalarını kaçış amaçlı kullanır.",
   
     "origin.toomanyorigins.swarm.name": "Swarm",
-    "origin.toomanyorigins.swarm.description": "Kovandan bağımsız olan bu böcekler kavgadan uzak duran uzman çiftçilerdir.",
+    "origin.toomanyorigins.swarm.description": "Swarm ırkı, çatışmadan uzak duran uzman çiftçi ve kovandan bağımsız böceklerdir.",
 
     "origin.toomanyorigins.undead.name": "Undead",
 
     "origin.toomanyorigins.withered.name": "Withered",
+    "origin.toomanyorigins.withered.description": "Withered ırkı, atalarının Wither'ın kaotik gücünden yararlanmalarının bir sonucu olarak öldürdüklerinin yaşam enerjisini emerler.",
   
-    "power.toomanyorigins.overheat.name": "Overheat",
-    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir objeden uzağa tepebilirsin. Bu işlem size hasar verir.",
+    "power.toomanyorigins.overheat.name": "Aşırı Isınma",
+    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni biraz incitir.",
```

Comment:
"rocket jump" as a term doesn't really exist in turkish so I translated the verb as "Perform a rocket jump" and not just "rocket jump"